helix-query.yaml

```diff
@@ -34,3 +34,32 @@ indices:
     include:
```

Comment:
@mayur-satav Can you take out the DE configuration from query as we no longer supporting it?

Reply:
only DE right ? or FR as well

Reply:
Both! Thank you for confirming!